tsconfig.tsbuildinfo

```diff
@@ -1 +1 @@
,[316,1],[315,1],[475,1],[474,1],[493,1],[492,1],[385,1],[384,1],[495,1],[494,1],[498,1],[496,1],[497,1],[500,1],[499,1],[580,1],[579,1],[503,1],[502,1],[501,1],[402,1],[401,1],[505,1],[504,1],[509,1],[508,1],[507,1],[506,1],[513,1],[512,1],[515,1],[514,1],[517,1],[516,1],[519,1],[518,1],[521,1],[520,1],[523,1],[522,1],[511,1],[510,1],[262,1],[264,1],[272,1],[574,1],[265,1],[586,1],[583,1],[362,1],[615,1],[587,1],[581,1],[582,1],[588,1],[575,1],[576,1],[591,1],[577,1],[589,1],[271,1],[590,1],[578,1],[298,1],[297,1],[525,1],[524,1],[527,1],[526,1],[529,1],[528,1],[531,1],[530,1],[533,1],[532,1],[535,1],[534,1],[537,1],[536,1],[539,1],[538,1],[541,1],[540,1],[544,1],[543,1],[542,1],[546,1],[545,1],[548,1],[547,1],[552,1],[551,1],[550,1],[549,1],[556,1],[555,1],[554,1],[553,1],[558,1],[557,1],[560,1],[559,1],[314,1],[364,1],[363,1],[562,1],[561,1],[564,1],[563,1],[566,1],[565,1],[294,1],[295,1],[299,1],[300,1],[301,1],[312,1],[302,1],[303,1],[304,1],[305,1],[306,1],[307,1],[308,1],[309,1],[310,1],[311,1],[570,1],[569,1],[568,1],[567,1],[572,1],[571,1],[592,1],[593,1],[584,1],[585,1],[268,1],[594,1],[595,1],[614,1],[597,1],[596,1],[599,1],[598,1],[601,1],[600,1],[266,1],[605,1],[604,1],[607,1],[606,1],[603,1],[602,1],[609,1],[608,1],[611,1],[610,1],[270,1],[269,1],[613,1],[612,1],[345,1],[344,1],[263,1],[158,1],[210,1],[682,1],[683,1],[616,1],[59,1],[60,1],[62,1],[63,1],[64,1],[65,1],[66,1],[67,1],[68,1],[69,1],[70,1],[71,1],[72,1],[74,1],[73,1],[75,1],[76,1],[77,1],[61,1],[111,1],[78,1],[79,1],[80,1],[112,1],[81,1],[82,1],[83,1],[84,1],[85,1],[86,1],[87,1],[88,1],[89,1],[90,1],[91,1],[92,1],[93,1],[95,1],[94,1],[96,1],[97,1],[98,1],[99,1],[100,1],[101,1],[102,1],[103,1],[104,1],[105,1],[106,1],[107,1],[108,1],[109,1],[110,1],[684,1],[50,1],[685,1],[687,1],[686,1],[688,1],[114,1],[689,1],[693,1],[690,1],[694,1],[692,1],[313,1],[691,1],[48,1],[52,1],[695,1],[51,1],[211,1],[233,1],[49,1],[199,1],[198,1],[197,1],[202,1],[201,1],[200,1],[223,1],[225,1],[227,1],[229,1],[241,1],[242,1],[244,1],[236,1],[239,1],[237,1],[235,1],[243,1],[245,1],[234,1],[240,1],[238,1],[267,1],[58,1],[168,1],[170,1],[128,1],[138,1],[129,1],[130,1],[139,1],[119,1],[179,1],[181,1],[163,1],[162,1],[161,1],[184,1],[160,1],[187,1],[115,1],[117,1],[167,1],[171,1],[175,1],[122,1],[121,1],[142,1],[150,1],[125,1],[120,1],[116,1],[154,1],[153,1],[143,1],[113,1],[151,1],[144,1],[152,1],[155,1],[131,1],[145,1],[126,1],[149,1],[148,1],[127,1],[141,1],[140,1],[132,1],[133,1],[135,1],[134,1],[136,1],[186,1],[137,1],[57,1],[165,1],[173,1],[177,1],[157,1],[118,1],[159,1],[164,1],[147,1],[146,1],[124,1],[123,1],[169,1],[166,1],[47,1],[56,1],[53,1],[54,1],[55,1],[172,1],[174,1],[176,1],[178,1],[191,1],[180,1],[190,1],[182,1],[183,1],[185,1],[188,1],[189,1],[156,1],[491,1],[2,1],[3,1],[4,1],[5,1],[6,1],[7,1],[8,1],[9,1],[666,1],[667,1],[668,1],[678,1],[680,1],[679,1],[681,1],[670,1],[671,1],[672,1],[673,1],[674,1],[675,1],[676,1],[677,1],[212,1],[213,1],[203,1],[231,1],[230,1]]},"version":"4.5.4"}
```

Comment:
@aaggarwal10 should we gitignore this?  I can't see what value it's providing to track.

Reply:
we definitely should gitignore the tsbuildinfo.